modules/CustomView/actions/Save.php

```diff
@@ -86,14 +86,14 @@ private function getCVModelFromRequest(\App\Request $request)
 			$selectedColumnsList = $defaultCustomViewModel->getSelectedFields();
 		}
 		$customViewData['columnslist'] = $selectedColumnsList;
-		$stdFilterList = $request->get('stdfilterlist');
-		if (!empty($stdFilterList)) {
-			$customViewData['stdfilterlist'] = $stdFilterList;
-		}
 		$advFilterList = $request->get('advfilterlist');
 		if (!empty($advFilterList)) {
 			$customViewData['advfilterlist'] = $advFilterList;
 		}
+		$duplicateFields = $request->get('duplicatefields');
```

Comment:
Make validation of the received data
```suggestion
		$duplicateFields = $request->get('duplicatefields',?);
```